src/components/VaccineData/VaccineDataMain.js

```diff
@@ -1,29 +1,14 @@
-import VaccineDataSingle from './SingleVaccineData/VaccineDataSingle';
-import CircularProgress from '@material-ui/core/CircularProgress';
+import VaccineDataSingle from "./SingleVaccineData/VaccineDataSingle";
 
 const VaccineDataMain = ({ vaccineData, loading }) => {
```

Comment:
Loading is decalred but not used? 
Why though?

Reply:
I need to fix the loading issue manually , so switched to Home!